### DISCUSSION TITLE
Get back

### DISCUSSION THREAD

**Discussion by Ekremaciloff** [blockscout/blockscout#8660] [Q&A]
How can I get back my tokens from browser?

**Comment by ekawibisono** [comment]
hello, I don't understand what you mean. explain in detail

**Comment by Ekremaciloff** [comment]
I sent my islm tokens from my haqq wallet to KuCoin account. But they didn't receive yet.

**Comment by ekawibisono** [comment]
Your question is off topic, please contact the Islamic Coin development team. Thank You
